Update notifications.mdx (#1713)
grommet/hpe-design-system: aries-site/src/pages/templates/notifications.mdx

* [Documenting accessibility guidelines](https://www.figma.com/file/FGvOd2htjrChVVS2oz73ds/HPE-Notifications?node-id=1%3A6)
* [Identifying potential user research opportunities](https://www.figma.com/file/FGvOd2htjrChVVS2oz73ds/HPE-Notifications?node-id=41%3A238)
* [Researching and reviewing literature and other external resources](https://www.figma.com/file/FGvOd2htjrChVVS2oz73ds/HPE-Notifications?node-id=25%3A238)

## Additional Resources

In additional to the exploration that the [HPE Design System](https://www.figma.com/file/FGvOd2htjrChVVS2oz73ds/HPE-Notifications?node-id=0%3A1) has done, you may find it helpful to view the research done by other teams on Notifications.

* [Common Cloud Service (CCS)](https://www.figma.com/file/vT3CaJANRAKHysuZs2JIGR/CCS-Notifications?node-id=1%3A17948)
* [Header Footer Service (HFS)](https://www.figma.com/file/AbxAC3KU59pPyHG5mnnh7R/HFS-North-Star?node-id=140%3A10059) 
* [Unified Customer Experience Platform (UCEP)](https://app.mural.co/t/hpecdoresearchteam7715/m/hpecdoresearchteam7715/1622038532199/60894a815784273763b6c9c3d71df4dc38fa8cb0?sender=uaae76a8060cc23bc06005320)

### Have more questions?

> Please be sure to send them over to the #hpe-design-system channel on slack!

Reviewer: ```suggestion
Please be sure to send them over to the `#hpe-design-system` channel on [Slack](https://slack-invite.grommet.io/)!
```

## Status

> Notifications is currently on the exploration and research phase.

Reviewer: ```suggestion
The Design System team is currently exploring and researching best practices and considerations for notification patterns. To stay up-to-date with the work being done, check out the links below:
```

> Stay in the loop and check out what is being worked on now:

Reviewer: ```suggestion
```

> * Defining the taxonomy

Reviewer: Can we link to specific frame of Figma documents where possible?

* Documenting accessibility guidelines
* Identifying potential user research opportunities
* Researching and reviewing literature and other external resources

> ### Resources

Reviewer: ```suggestion
### Additional Resources
```

> Aside from what the [design system](https://www.figma.com/file/FGvOd2htjrChVVS2oz73ds/HPE-Notifications?node-id=0%3A1) has done, be sure to view the work done by other teams on Notifications:

Reviewer: ```suggestion
In additional to the exploration that the [HPE Design System](https://www.figma.com/file/FGvOd2htjrChVVS2oz73ds/HPE-Notifications?node-id=0%3A1) has done, you may find it helpful to view the research done by other teams on Notifications.
```

* [Header Footer Service (HFS)](https://www.figma.com/file/AbxAC3KU59pPyHG5mnnh7R/HFS-North-Star?node-id=140%3A10059) 

> * [Unified Customer Experience Platform (UCEP)](https://www.figma.com/file/AbxAC3KU59pPyHG5mnnh7R/HFS-North-Star?node-id=140%3A10059)

Reviewer: This link needs to be updated to the UCEP link

> * [Common Cloud Service (CCS)](https://www.figma.com/file/vT3CaJANRAKHysuZs2JIGR/CCS-Notifications?node-id=1%3A17948)

Reviewer: Can we put this link first?
